src/metabase/events/schema.clj

```diff
@@ -0,0 +1,133 @@
+(ns metabase.events.schema
```

Comment:
For a follow-up PR: Rather than centralize event schemas in one place and have all this code to merge them together, I suggest dropping the schemas and thinking about how to create a multimethod like `validate-event` that is dispatched by event topic. 

Imagine if all our functions has schemas in one namespace! We'd waste a lot of our dev time just switching between files and following references

Reply:
see my other [comment](https://github.com/metabase/metabase/pull/34947/files#r1375607002) for details on why I chose this approach.

Reply:
Still, it feels desirable to change `events.schema/topic->schema` to be a multimethod so you don't have to merge all the schemas into a map and you can define them in any namespace.